content/components/only-manipulate-the-dom-via-the-renderer.md

```diff
@@ -0,0 +1,50 @@
+---
+title: only manipulate the DOM via the Renderer
+author: 
+  name: Billy Lando (@billyjov)
+  url: https://github.com/billyjov
+---
+
+# Problem 
+
+According to the angular documentation, relying on direct DOM access creates tight coupling between your application and rendering layers which will make it impossible to separate the two and deploy your application into a web worker. Permitting direct access to the DOM can make your application more vulnerable to **XSS** attacks.
```

Comment:
`angular` => `Angular`

Great! I would also like to mention that it's not recommended to use the `document` object for DOM manipulations or to query for DOM elements. I have seen this a couple of times when doing code reviews.

Also, using `document` will not work when using AoT as the `document` object is not available on other platforms and it's better to use abstractions like the `Renderer2`. AoT is pretty important here too.